airbyte-cdk/python/airbyte_cdk/test/entrypoint_wrapper.py

```diff
@@ -36,7 +36,7 @@
     TraceType,
     Type,
 )
-from pydantic.error_wrappers import ValidationError
+from pydantic.v1.error_wrappers import ValidationError
```

Comment:
This is used to parse a JSON string as an `AirbyteMessage`. From the PR description, I was assuming that `airbyte_protocol.models` (which `AirbyteMessage` is part of) would use Pydantic V2. Hence, shouldn't we have the `ValidationError` imported from the V2 package?

Reply:
I think you're right - this is a good catch! I'll add a test to be sure, thank you

Reply:
You were right - added a test that fails on the v1 import here [b16de58](https://github.com/airbytehq/airbyte/pull/39524/commits/b16de582281b92f94385185672bda9870557f4e1)